src/utils/spark.py

```diff
@@ -17,8 +17,6 @@
     strip_table_prefix,
 )
 
-logger = create_python_logger(__name__)
-
```

Comment:
There's no reason why we couldn't _also_ define this logger on a module level and use it in code outside of the `SparkJob` class, but since `SparkJob` is the only code that logs right now, I'm removing this logger definition until we decide we need it.